### ISSUE TITLE
Error on https://uploads.ungrounded.net/ruffle_wrapper/ruffleembed.html

### ISSUE THREAD

**Issue by MrDarkEssense** [ruffle-rs/ruffle#7978]
# Error Info
Error name: Error
Error message: panicked at 'already mutably borrowed: BorrowError', core/src/display_object/bitmap.rs:261:44
Error stack:
```
Error: panicked at 'already mutably borrowed: BorrowError', core/src/display_object/bitmap.rs:261:44
    at W.n.wbg.__wbg_new_df6e6ab7a65c4c4d (https://uploads.ungrounded.net/ruffle_wrapper/lib/core.ruffle.b6ae0fd3c86ec8102bb0.js:1:5314)
    at std::sync::once::Once::call_once::{{closure}}::hc7e1e9ccae1dc751 (wasm://wasm/0178a3d2:wasm-function[1137]:0x2be2f6)
    at std::sync::once::Once::call_inner::h00971925aa1adad3 (wasm://wasm/0178a3d2:wasm-function[1438]:0x314076)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::heaafb58166fe27c7 (wasm://wasm/0178a3d2:wasm-function[4836]:0x46f693)
    at std::panicking::rust_panic_with_hook::h84feca33bd4bd229 (wasm://wasm/0178a3d2:wasm-function[3600]:0x4378e5)
    at core::panicking::panic_fmt::hc171d095bc4a492d (wasm://wasm/0178a3d2:wasm-function[5023]:0x475db7)
    at core::result::unwrap_failed::h68ab818eb89182b6 (wasm://wasm/0178a3d2:wasm-function[4191]:0x45b815)
    at <ruffle_core::display_object::DisplayObject as ruffle_core::display_object::TDisplayObject>::render_self::hd31e2ebd3082acc3 (wasm://wasm/0178a3d2:wasm-function[2506]:0x3d8be6)
    at ruffle_core::display_object::render_base::h5074f75d49260521 (wasm://wasm/0178a3d2:wasm-function[1158]:0x2c3eb3)
    at <ruffle_core::display_object::DisplayObject as ruffle_core::display_object::TDisplayObject>::render::h0fd7fdd56d756010 (wasm://wasm/0178a3d2:wasm-function[3821]:0x446b3a)
```

# Player Info
Allows script access: true
Renderer: WebGL

# Page Info
Page URL: https://uploads.ungrounded.net/ruffle_wrapper/ruffleembed.html?v=1.0.67&useloader=1&browsermode=default&props=%7B%22ruffle%22%3A%22%2Fruffle_wrapper%2Flib%2Fruffle.js%3F1663078315%22%2C%22public_path%22%3A%22%2Fruffle_wrapper%2Flib%2F%22%2C%22swf%22%3A%22https%3A%2F%2Fuploads.ungrounded.net%2F592000%2F592473_MadnessGame_ULTIMATE.swf%3F1507662264%22%2C%22vars%22%3A%7B%22ngio_session_id%22%3A%2291343407.23e686e1171236d3da8856f8aabccddffa24c476afc796%22%2C%22NewgroundsAPI_PublisherID%22%3A1%2C%22NewgroundsAPI_SandboxID%22%3A%226321711e6a329%22%2C%22NewgroundsAPI_SessionID%22%3A%22XeKOCG5OX84Pfr7rAjqs763f78ee370177bda5937c7ff652c92bff8efbcbe982%22%2C%22NewgroundsAPI_UserName%22%3A%22dumBabe%22%2C%22NewgroundsAPI_UserID%22%3A10750373%2C%22ng_username%22%3A%22dumBabe%22%7D%2C%22width%22%3A850%2C%22height%22%3A530%2C%22icon%22%3A%22https%3A%2F%2Fpicon.ngfiles.com%2F592000%2Fflash_592473_card.png%3Ff1608170534%22%2C%22warnOnUnsupportedContent%22%3Afalse%7D
SWF URL: https://uploads.ungrounded.net/592000/592473_MadnessGame_ULTIMATE.swf?1507662264

# Browser Info
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36
Platform: Win32
Has touch support: false

# Ruffle Info
Version: 0.1.0
Name: nightly 2022-09-13
Channel: nightly
Built: 2022-09-13T00:14:48.844Z
Commit: bff0855435987bb9f19f48e120ad7cbb77f9d924
Is extension: false

# Metadata
width: 850
height: 530
frameRate: 30
numFrames: 30
swfVersion: 10
backgroundColor: #FFFFFF
isActionScript3: false
uncompressedLength: 20491352

**Comment by n0samu** [comment]
Here are the steps to reproduce this issue:
1. Go to the *Madness: Project Nexus (Classic)* page: https://www.newgrounds.com/portal/view/592473
2. Get into the game by clicking Story Mode => New Game => Click the jail building => Proceed
3. Play through the tutorial into the red "Bullet-Time" bar across the top fills up
4. Press the Shift key to activate Bullet-Time. The game will crash.

**Comment by n0samu** [comment]
@sussybakatea123 is reporting some additional problems with the game's Arena Mode:
- Cannot see dashes `-` and underscores `_` in the text
- After you die, the game freezes

**Comment by n0samu** [comment]
Fixed by #8248
